MAINTAINERS.yml

```
@@ -2362,6 +2362,17 @@ Storage:
   labels:
     - "area: Storage"
 
+Sysbuild:
+  status: maintained
+  maintainers:
+    - tejlmand
+  collaborators:
+    - nordicjm
+  files:
+    - share/sysbuild/
```

would be nice to include relevant sample and test directories as well.

```suggestion
    - share/sysbuild/
    - samples/application_development/sysbuild/
```

---

Added to MAINTAINERS and CODEOWNERS